src/App/Command/Git/StatusCommand.php

```diff
@@ -30,14 +30,17 @@ protected function processPackage(Package $package): void
         $io = $this->getIO();
         $io->preparePackageHeader($package, 'Git status of {package}');
 
+        $branchName = $package->getGitWorkingCopy()->getBranches()->head();
+
         $process = new Process(['git', 'status', '-s'], $package->getPath());
         $process->run();
         $output = $process->getOutput();
 
         if (empty($output)) {
-            $io->success('✔ nothing to commit, working tree clean');
+            $io->important()->success("[$branchName] ✔ nothing to commit, working tree clean");
```

Comment:
(Optional, code style) Invert to keep the shorter block under `if` and longer - under `else`. But with that size not critical at all.